### ISSUE TITLE
Confusing prompt: "Enter password for False"

### ISSUE THREAD

**Issue by lpsinger** [duncanmmacleod/requests-ecp#3]
I get the confusing message "Enter password for False" when I try to authenticate against ligo.org:

```pycon
>>> import requests
>>> from requests_ecp import HTTPECPAuth
>>> auth = HTTPECPAuth('https://login.ligo.org/idp/profile/SAML2/SOAP/ECP')
Enter password for False on login.ligo.org: 
```

**Comment by duncanmmacleod** [comment]
This was fixed by #1.

**Comment by duncanmmacleod** [comment]
In the mean time, just make sure and pass `username="leo.singer"` as a keyword.
